platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -279,7 +281,8 @@ def parse_onie_version(self, version, is_base=False):
         onie_major = m.group(3)
         onie_minor = m.group(4)
         onie_release = m.group(5)
-        onie_baudrate = m.group(6)
+        onie_signtype = m.group(6)
```

Comment:
@DavidZagury how about backward compatibility? Not all ONiE FW will have dev in version string